edu_edfi_airflow/callables/airflow_util.py

```diff
@@ -121,17 +132,35 @@ def get_snowflake_params_from_conn(
         raise undefined_snowflake_error
 
 
-def subset_kwargs_to_class(class_: object, kwargs: dict) -> dict:
+def fail_if_any_task_failed(**context):
+    """
+    Simple Python callable that raises an AirflowFailException if any task in the DAG has failed.
+    """
+    for ti in context["dag_run"].get_task_instances():
+        if ti.state == "failed":
+            raise AirflowFailException("One or more tasks in the DAG failed.")
+
+
+def chain_tasks(*tasks):
     """
-    Helper function to remove unexpected arguments from kwargs,
-    based on the actual arguments of the class.
+    Alias of airflow's built-in chain, but remove Nones if present.
+    Note: this recurses only one level.
```

Comment:
I think this note is no longer true

Reply:
You're very correct. Building the `recursive_filter()` helper took a couple passes, and I wasn't sure it would be done cleanly. I'll remove this docstring comment.